MIGRATION.md

```diff
@@ -0,0 +1,35 @@
+# Migration guide
+
+## From 0.x.x to 1.0.0
+
+On the backend side the changes are simple:
+
+1. Remove the `AsyncRequest::ApplicationHelper` from your controller.
+2. The gem now does not expect a JSON as the result of the worker but any a string, so you now must perform a `.to_json` to the result of the worker:
+```ruby
+class MyComputeHeavyWorker
+  def execute(options, string)
+    # Perform heavy task
+    [:ok, { message: 'success' }.to_json]
+  end
+end
+```
+3. Where you were doing something like `execute_async(MyComputeHeavyWorker, arg)` now you have to do `AsyncRequest::Job.create_and_enqueue(MyComputeHeavyWorker, arg)`. This will return the new `AsyncRequest::Job` instance.
```

Comment:
I don't like very much how this is explained, the phrase `Where you were doing something like` sounds way to informal for me